game_board/src/fr/lecampusnumerique/cda2025/javaalgo/boardgames/games/GomokuGame.java

```diff
@@ -0,0 +1,7 @@
+package fr.lecampusnumerique.cda2025.javaalgo.boardgames.games;
+
+public class GomokuGame extends AbstractGame {
+    public GomokuGame() {
+        super(15, 15);
```

Comment:
what is serve: super (15, 15). u can comment it next time?

Reply:
super means I am using the constructor of the parent component :) the parent component, in order to build its children, require the amount of rows and columns :) this don't need to be commented as it is a classic functionality of Java (meaning I didnt invent it nor built it myself)